testing/batch_processing/cached_results/dmri/fa_in_cst.csv

```diff
@@ -1,3 +1,3 @@
 Timestamp,SCT Version,Filename,Slice (I->S),VertLevel,Label,Size [vox],WA(),STD()
-2021-03-16 10:49:07,git-jn/3254-fix-sform-sct_warp_template-9566dce5850797d73d819917f7f49ce74b4a052c,/home/joshua/repos/spinalcordtoolbox/sct_example_data/dmri/dti_FA.nii.gz,2:14,,WM left lateral corticospinal tract,70.35704290421518,0.7793999429926204,0.13533664759850947
-2021-03-16 10:49:07,git-jn/3254-fix-sform-sct_warp_template-9566dce5850797d73d819917f7f49ce74b4a052c,/home/joshua/repos/spinalcordtoolbox/sct_example_data/dmri/dti_FA.nii.gz,2:14,,WM right lateral corticospinal tract,71.21054427612819,0.7755359256476565,0.13641569050448102
+2021-03-16 10:49:07,git-jn/3254-fix-sform-sct_warp_template-9566dce5850797d73d819917f7f49ce74b4a052c,/home/joshua/repos/spinalcordtoolbox/sct_example_data/dmri/dti_FA.nii.gz,2:14,,WM left lateral corticospinal tract,70.35704290421518,0.7793405440192693,0.13533664759850947
+2021-03-16 10:49:07,git-jn/3254-fix-sform-sct_warp_template-9566dce5850797d73d819917f7f49ce74b4a052c,/home/joshua/repos/spinalcordtoolbox/sct_example_data/dmri/dti_FA.nii.gz,2:14,,WM right lateral corticospinal tract,71.21054427612819,0.7755611783165834,0.13641569050448102
```

Comment:
The ground truth values here seem to be incorrect/flawed in much the same way that the `sct_dmri_moco` ground truth values are off.

I'm going to try to dig a little bit more into what exactly is causing the values to change.